package.json

```diff
@@ -1,28 +1,28 @@
 {
   "name": "@tinymce/eslint-plugin",
-  "version": "1.4.1",
+  "version": "1.5.0",
   "description": "Plugin for ESLint containing Tiny Technologies Inc. standard linting rules",
   "author": "Tiny Technologies Inc.",
   "main": "dist/main/ts/api/Main.js",
   "repository": "git@github.com:tinymce/eslint-plugin.git",
   "license": "Apache-2.0",
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.29.0",
-    "@typescript-eslint/parser": "^2.29.0",
-    "eslint": "^6.8.0",
-    "eslint-plugin-import": "^2.20.1",
-    "eslint-plugin-prefer-arrow": "^1.1.7",
-    "resolve": "^1.10.0",
-    "typescript": "^3.8.0"
+    "@typescript-eslint/eslint-plugin": "^4.3.0",
+    "@typescript-eslint/parser": "^4.3.0",
+    "eslint": "^7.10.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-prefer-arrow": "^1.2.2",
+    "resolve": "^1.17.0",
+    "typescript": "^4.0.3"
```

Comment:
Is this dependency correct? As far as I can see it doesn't require 4.0.3 specifically? Should this actually be a peer or even a dev dependency?

Reply:
I set it back to ^4